### DISCUSSION TITLE
Playtime: Separate button to load instrument with instant opening of search window instead of 2 separate windows (FX chain add window -> then Plugin Search window)

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1249] [Ideas]
When trying to add new instrument it quite kills the creative spirit by having to open 1st window of FX Chain and then to cick ADD and then the 2nd window pops up to search for instrument, would it be possible to have a separate icon for instrument that instantly pops up the ‘search window of plugins’ instead of 2 windows? Let’s say for instrument I would leave ‘Icon of a note’ as seen in the picture below, and for effects (and FX chain) I would leave the icon of ‘FX’ in the middle. When a person creates it really makes a big difference to load sounds fast and quick without going through a kind of bureaucracy. 
![image](https://github.com/user-attachments/assets/b509b0a9-1aea-4ffa-90b7-5b1447bf82da)

**Comment by helgoboss** [comment]
Could be nice. Another option would be #975?

**Reply by joungle123** [reply]
@helgoboss Reaper allows for creating default track templates, so that should load automatically in Playtime 2? I am my not right? (this is regarding #975 request

**Reply by joungle123** [reply]
Regarding #975 suggestion - I mean you could create a default track template for reaper and when Playtime creates new track then that default reaper track would load as created. Am I wrong here? I think #975 request is not required

**Reply by helgoboss** [reply]
Don't know, never tried default tracks.

But #975 is not about default tracks. It's about being able to select already existing FX chains and track templates directly from the Playtime UI.

**Reply by joungle123** [reply]
@helgoboss Oh I get it know, yes it is a good idea as well! Not sure about FX chains as I only use it on the mix bus but for track templates it would be nice

**Comment by jimmydeer** [comment]
Instead of having two buttons, would it be easier (since dev hasn't much time) to make the one button open the actual fx library? Idk if that was suggested but I thought I saw it somewhere and was looking upvote. When I click on an insert slot in reaper-everything opens, this would be great
<img width="1920" alt="Screenshot 2025-06-08 at 5 22 58 PM" src="https://github.com/user-attachments/assets/275d9cf4-dfa3-456f-955a-8a6abc34b555" />




+1 for selecting templates/chains
